docs/Community/sig-architecture/OWNERS

```diff
@@ -0,0 +1,17 @@
+chair:
+  - TBD
+
+# reviews should be solicited from all SIG Chairs
+# NOTE(Aeva) missing domains: API, CCF, Windows
+reviewers:
+  - sig-attestation-chair
+  - sig-testing-chair
+  - sig-tools-chair
+  - sig-release-chair
+  - sig-sgx-chair
+  - sig-optee-chair
```

Comment:
After reviewing the docs, I think it makes sense to have a SIG-API.

Reply:
Maybe document why website/docs are not on the list?

Reply:
added sig-api (empty list for now) and a note about sig-website